src/components/TodoList/TodoList.tsx

```diff
@@ -16,7 +32,50 @@ export const TodoList: React.FC = () => (
     </thead>
 
     <tbody>
-      <tr data-cy="todo" className="">
+      {todos.map(todo => (
+        <tr data-cy="todo" className="" key={todo.id}>
+          <td className="is-vcentered">{todo.id}</td>
+          <td className="is-vcentered">
+            {todo.completed && (
+              <span className="icon" data-cy="iconCompleted">
+                <i className="fas fa-check" />
+              </span>
+            )}
+          </td>
+          <td className="is-vcentered is-expanded">
+            <p
+              className={cn({
+                'has-text-danger': !todo.completed,
+                'has-text-success': todo.completed,
+              })}
+            >
+              {todo.title}
+            </p>
+          </td>
+          <td className="has-text-right is-vcentered">
+            <button
+              data-cy="selectButton"
+              className="button"
+              type="button"
+              onClick={() => {
+                setShowModal(true);
+                setTodo(todo);
+                setIconEyeId(todo.id);
+              }}
+            >
+              <span className="icon">
+                <i
+                  className={cn('far', {
+                    'fa-eye': todo.id !== iconEyeId,
+                    'fa-eye-slash': todo.id === iconEyeId,
+                  })}
```

Comment:
Ensure that the `classnames` library is correctly managing the dynamic classes for the icons. Verify that the logic for toggling between 'fa-eye' and 'fa-eye-slash' is working as intended.